packages/foam-vscode/src/features/orphans.ts

```diff
@@ -19,9 +13,12 @@ const feature: FoamFeature = {
     const workspacesFsPaths = vscode.workspace.workspaceFolders.map(
       dir => dir.uri.fsPath
     );
-    const provider = new OrphansProvider(
+    const provider = new FilteredNotesProvider(
       foam.workspace,
       foam.services.dataStore,
+      'orphans',
+      'orphan',
+      (note: Note) => isOrphan(foam.workspace, note),
       {
         ...getOrphansConfig(),
         workspacesFsPaths,
```

Comment:
This is the real "secret sauce" of this change. We no longer have a specific `OrphansProvider`, but rather we use a generic `FilteredNotesProvider` which provides all of the same methods, but requires the called to provide a filter function. In the case of the `Orphans` feature, that filter is:
```typescript
export function isOrphan(workspace: FoamWorkspace, note: Note) {
  return workspace.getConnections(note.uri).length === 0;
}
```